sdk/src/main/java/com/qonversion/android/sdk/internal/di/controllers/ControllersAssemblyImpl.kt

```diff
@@ -39,6 +43,10 @@ internal class ControllersAssemblyImpl(
             miscAssembly.logger()
         )
 
+    override fun userController(): UserController = UserControllerImpl(
+        servicesAssembly.userService(), cacherAssembly.userCacher(), miscAssembly.logger()
```

Comment:
For enumeration I prefer one line per argument syntax. And you? 
```suggestion
        servicesAssembly.userService(),
        cacherAssembly.userCacher(),
        miscAssembly.logger()
```